### DISCUSSION TITLE
I only want windows platform project how to create a new?

### DISCUSSION THREAD

**Discussion by JHeLiu** [dotnet/maui#14735] [Q&A]
I only want windows platform project how to create a new?

![image](https://user-images.githubusercontent.com/15246138/233949443-5f99907f-cebe-4136-b567-ad87a05ea890.png)

I need this.
![image](https://user-images.githubusercontent.com/15246138/233949549-ce6b95b5-dcad-446e-a1f4-052920206002.png)

**Comment by KarthikHebbarMN** [comment]
If you just want Windows. Use WPF, it's a similar to MAUI have been for longer than MAUI so its much more stable and its specifically built for Windows.
